### PR TITLE
Feature/multi language

### PR DESCRIPTION
**Pull request by dhapin**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (security config, docs update, etc.)

## Screenshot of change (if there's UI update)

### CODE REVIEW

**Comment by wildan3105** [comment]
Thanks for the PR! Notes:

Please fetch the latest master and also fix the failed build. And because there's no button for user to switch the language, I think it's not ready to merge this onto master. I'll create intermediary target branch for this.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-langs",
-  "version": "2.0.16",
```
Comment (wildan3105) [inline package.json:3]: Please don't downgrade the version

File: package.json [hunk 2]
```diff
@@ -92,15 +92,15 @@
     "gulp": "^4.0.0",
     "gulp-rename": "^2.0.0",
     "gulp-sass": "^3.1.0",
+    "i18n": "^0.13.2",
     "lodash": "^4.17.19",
     "method-override": "^2.3.5",
     "nodemon": "^1.19.3"
   },
   "devDependencies": {
-    "eslint": "~4.8.0",
```
Comment (wildan3105) [inline package.json:100]: This one also, do fetch latest master and accept incoming change.

File: views/partials/head.handlebars [hunk 3]
```diff
@@ -19,11 +19,11 @@
   <meta property="og:image" content="https://githublangs.herokuapp.com/public/img/favicon.png" />
 
   <link rel="icon" href="public/img/favicon.png">
-  <title>Github Language Checker {{ title }}</title>
```
Comment (wildan3105) [inline views/partials/head.handlebars:22]: Sorry, no need to translate the page title

**Review by wildan3105 — CHANGES_REQUESTED** [review]
Please address my feedbacks

File: package.json (hunk 1 above)
Comment (dhapin) [reply package.json:3]: Sorry, I didn't know that the code was updated because I fetched the code this morning.

Comment (wildan3105) [reply package.json:3]: ok please refetch again

Comment (dhapin) [reply package.json:3]: 👍

File: lib/locales/en.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+	"title": "GitHub Language Checker",
+	"back": "Back to home",
+	"username.placeholder": "Github username", 
+	"button.color": "Toggle Color",
+	"button.download": "Download", 
+    "button.share": "Bagikan",
```
Comment (wildan3105) [inline lib/locales/en.json:7]: this one should be `Share`, right?

Comment (dhapin) [reply lib/locales/en.json:7]: Yes, right

**Pull request closed without merging**